pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -22,6 +22,7 @@
 import static org.apache.commons.lang3.StringUtils.isBlank;
 import static org.apache.pulsar.broker.BrokerTestUtil.newUniqueName;
 import static org.apache.pulsar.broker.resources.LoadBalanceResources.BUNDLE_DATA_BASE_PATH;
+import static org.apache.pulsar.common.policies.data.NamespaceIsolationPolicyUnloadScope.*;
```

Comment:
Please avoid star imports

Reply:
Any reason why a star import is bad here? there can't be any conflict as the enum would only be usable in a place where it is expecting an enum of class `NamespaceIsolationPolicyUnloadScope`.

Reply:
@grssam The main reason is that Pulsar code base uses such a code style. You get the correct formatting in IntelliJ if you apply the Pulsar code style as instructed in https://pulsar.apache.org/contribute/setup-ide/.

Reply:
@dao-jun Thank you for the review.. but since this is an enum import and i am using all of the enums in the tests.. I'll leave it as is.
I do understand normal class imports via star being harmful to future naming conflicts, but this one shouldn't have any even in future.